drivers/rtc/rtc_ll_stm32.c

```diff
@@ -29,6 +29,10 @@
 
 LOG_MODULE_REGISTER(rtc_stm32, CONFIG_RTC_LOG_LEVEL);
 
+/* Whether or not the hardware support subsecond counting */
```

Comment:
Not related with the PR description. Please update for clarity

Reply:
This was to fix CI build which was failing while trying to build for the stm32l1_disco.

Reply:
I just realized you where probably talking about the PR description itself. This is now done.